openrndr-shape/src/main/kotlin/org/openrndr/shape/Rectangle.kt

```diff
@@ -78,6 +79,14 @@ data class Rectangle(val corner: Vector2, val width: Double, val height: Double)
         fun fromCenter(center: Vector2, width: Double, height: Double) =
                 Rectangle(center.x - width / 2.0, center.y - height / 2.0, width, height)
     }
+
+    fun toIntRectangle(): IntRectangle {
```

Comment:
I'm unsure about this method.. @edwinRNDR do we have a pattern for conversions between classes?